Silicon/CommonSocPkg/Library/PchSpiLib/PchSpiLib.c

```diff
@@ -83,6 +94,13 @@ EnableBiosWriteProtect (
   IN  UINTN         PchSpiBase
   )
 {
+  UINT8   BiosCtrl;
+
+  // Set the BIOS Lock Enable and EISS bits
+  BiosCtrl = MmioRead8 (PchSpiBase + R_SPI_BCR);
+  if (((BiosCtrl & B_SPI_BCR_SMM_BWP) != 0) && ((BiosCtrl & B_SPI_BCR_BLE) == 0)) {
+    MmioOr8(PchSpiBase + R_SPI_BCR, (UINT8)(B_SPI_BCR_SMM_BWP));
+  }

```

Comment:
Here only EISS is set, it doesn't match with the comments "Set the BIOS Lock Enable and EISS bits"
Please update the comments.

Reply:
I've updated the comment here. Thanks @gdong1

Reply:
Update B_SPI_BCR_SMM_BWP to B_SPI_BCR_EISS in this patch.

Reply:
And the if condition should be BiosCtrl & B_SPI_BCR_SMM_BWP) == 0 instead of "!=0".

Reply:
Done.